### DISCUSSION TITLE
Error Connecting after self-hosted

### DISCUSSION THREAD

**Discussion by crapthings** [nhost/nhost#1801] [Q&A]
ubuntu 20

https://github.com/nhost/nhost/tree/main/examples/docker-compose

```
Error Connecting
Did you forget to start nhost up? Please refer to the [CLI documentation](https://docs.nhost.io/platform/cli) if you are having trouble starting your project.
```

i've followed the self-hosted doc, and something wrong with it.

i expect dashboard to be shown like cloud one

**Comment by R3flector** [comment]
Same error
